### ISSUE TITLE
Ascii output format

### ISSUE THREAD

**Issue by danielsnider** [tone-row/flowchart-fun#154]
Hello again 😊

I made [a chart](https://flowchart.fun/nsys-to-df) to help document how some software works. I just wish I could generate ascii of the flow chart so that I can paste the chart within the code. Probably a tall order, eh?

Great project,
Thanks!

**Comment by rob-gordon** [comment]
Hey @danielsnider ! I don't have access to your chart via the link you provided. Could you click `Share` and copy and paste the long share url.

Just to clarify, you want to be able to export your graph as all ASCII text. Something like this?
```
┌───────────┐
│hello world│
│           ├───┐
│           │   │
└───────────┘   │
                │
                ▼
```

**Comment by danielsnider** [comment]
Yes, that is a really nice example of output as ascii text! Exactly what I am thinking of.
My graph is not that interesting but here it the [working URL](https://flowchart.fun/c#H4hQBsEME8HsFcAuAuUACNA7SBbApsmgCaQDmATnumuZJgNYAiAluYQDIBKoIwomAZ2gCABQAoBiPAAcAvAEYAlKBEiBAR3DMpK1SKIAzXXvEBjWDgBGzTHiL6DywcPGSZsgEzK1m7VVWGxmYW1rb2hk5CohJScpjeGlo6AUYmYuZWNnYOyhmh2YEpAPqIsEXS4LCIoEA).

**Comment by beejei** [comment]
I could find some examples that provide a way to create ASCII diagram from scratch [here](https://xosh.org/text-to-diagram/), but exporting to that (specific) format is totally a new story.
